packages/ppx/src/Property_to_runtime.re

```diff
@@ -4462,7 +4489,21 @@ let scrollbar_base_color =
   unsupportedProperty(Property_parser.property_scrollbar_base_color);
 
 let scrollbar_color =
-  unsupportedProperty(Property_parser.property_scrollbar_color);
+  monomorphic(
+    Property_parser.property_scrollbar_color,
+    (~loc) => [%expr CSS.scrollbarColor],
+    (~loc, value: Types.property_scrollbar_color) =>
+      switch (value) {
+      | `Auto => [%expr `auto]
+      | `Color([thumbColor, trackColor]) =>
+        [%expr
+         `thumbTrackColor((
+           [%e render_color(~loc, thumbColor)],
+           [%e render_color(~loc, trackColor)],
+         ))]
+      | `Color(_) => raise(Unsupported_feature)
```

Comment:
one color looks like isn't supported (can we change the spec definition to be <color> <color> instead?

Reply:
this path will not be reached i believe, since we are using the curly syntax ``<color>{2}`` the type will always be a list

Reply:
Can you add a comment (it's easy to discover in the future but faster with a small //)